src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -289,6 +290,36 @@ public class EntityFrameworkServices : IEntityFrameworkService
             return true;
         }
 
+        // Look for the model type in the current project. 
+        // If its not found in the current project, look in the dependencies.
+        private Type GetModelTypeFromAssembly(string modelTypeName, Assembly assembly)
+        {
+            Type modelType = assembly.GetType(modelTypeName);
+            if (modelType == null)
+            {
+                // Need to look in the dependencies of this project now.
+                var dependencies = assembly.GetReferencedAssemblies().GetEnumerator();
```

Comment:
At this point a build has already happened. 
Also, before reaching here, we already make sure that the modelType actually exists using the ModelTypesLocator. 

In this particular part, we are trying to get the modelType from the in-memory compilation of the project (and its dependencies) after the DbContext has been added/ modified.